charts/authelia/values.yaml

```diff
@@ -1407,7 +1413,7 @@ configMap:
     ## Important: Kubernetes (or HA) users must read https://www.authelia.com/overview/authorization/statelessness/
     ##
     filesystem:
-      enabled: false
+      enabled: true
```

Comment:
```suggestion
      enabled: false
```

Reply:
ok, in that case, the lint requirements needs to be  changed, as the lint is looking for one of these to be true, same as for all the changes I made in this commit.

Reply:
Yeah I figured that may be the case. The problem is when you set booleans as true in a default values.yaml depending on various factors it can be really hard to override them in certain scenarios. This leads to a very unfavorable user experience since they cant un-configure something. 

In addition users are required to set local values in the chart and the validations are a good indication they didn't.

Reply:
Ok, I have figured out the lint, instead of looking for enabled, it will look for the existance of the key eg "filesystem" I will commit soon soon.

Reply:
I also have been debugging the odic client_secret, at the moment its failing on the hash checks